CHANGELOG.md

```diff
@@ -29,6 +29,47 @@ You can check your current version with the following command:
     ```
 
 For more information, see [UP42 Python package description](https://pypi.org/project/up42-py/).
+## 1.0.4
+
+**Jun 17, 2024**
+
+**New Features**
```

Comment:
Let's use ## instead of ** for headers and ### for sub headers instead of `**-` and `:**`